### ISSUE TITLE
dbus: Cannot receive messages after sending a message

### ISSUE THREAD

**Issue by EmmanuelIT** [AsamK/signal-cli#312]
I am trying to send and receive messages through dbus using pydbus Python library.

What I discovered is that if I first send a message to a recipient, I can no longer receive messages from the system bus unless I call signal.sendEndSessionMessage(...) for the same recipient.

Steps to reproduce the issue:

1. Send the message through dbus using **sendmessage.py**
```python
from pydbus import SystemBus
import os

bus = SystemBus()
signal = bus.get('org.asamk.Signal')

signal.sendMessage("one message", [], ['+991234567890'])
```
2. Receive messages from dbus using **rcvmessages.py**
```python
from pydbus import SystemBus
from gi.repository import GLib

def msgRcv (timestamp, source, groupID, message, attachments):
   print("onMessageReceived:", timestamp, source, groupID, message, attachments)
   return

bus = SystemBus()
loop = GLib.MainLoop()

signal = bus.get('org.asamk.Signal')
signal.onMessageReceived = msgRcv
loop.run()
```
At this point, no messages are received and msgRcv() is never called.
Doing a **tail -f /val/log/syslog** at this point, when a message arrives, you get:

```log
May 23 13:04:00 UBUPC signal-cli[11612]: Envelope from: unknown source
May 23 13:04:00 UBUPC signal-cli[11612]: Timestamp: 1590228240459 (2020-05-23T10:04:00.459Z)
May 23 13:04:00 UBUPC signal-cli[11612]: Sent by unidentified/sealed sender
May 23 13:04:00 UBUPC signal-cli[11612]: Exception: org.whispersystems.libsignal.InvalidMessageException: No valid sessions. (ProtocolInvalidMessageException)
May 23 13:04:00 UBUPC signal-cli[11612]: Failed to decrypt message.
```

The only workaround I found, was to call
```python
signal.sendEndSessionMessage(['+991234567890'])
```
imediately after signal.sendMessage(...) call.

But this will generate a "Secure session reset" on signal, which is of course an unwanted behavior. 

Can you please help me with this problem ?

*PS: If you run **rcvmessages.py** without first running **sendmessage.py** it works perfectly, with messages flowing through msgRcv(...) callback.*

**Comment by AsamK** [comment]
Duplicate of #309 .
Should be fixed already in version 0.6.8.

**Comment by bcashlol** [comment]
Using v0.6.8 and desktop signal 1.35.5, I have a very similar (or the same) issue.

> [R]Envelope from: “null” +11234567890 (device: 6)
Timestamp: 1597330822040 (2020-08-13T14:54:43.080Z)
Exception: org.whispersystems.libsignal.InvalidMessageException: No valid sessions. (ProtocolInvalidMessageException)
Failed to decrypt message.

_(modified the phone number for privacy)_
